docker-bench-security.sh

```diff
@@ -19,6 +19,7 @@ dir_name=`dirname ${this_path}`    ## Dir where this file is
 myname=`basename ${this_path}`     ## file name of this script.
 logger="${myname}.log"
 
+export PATH=/bin:/sbin:/usr/bin
```

Comment:
A simple way to reduce risk of the script misbehaving. Some sort of response to Issue #2.